### PR TITLE
fix: Scales onboarding slides to fit smaller windows

### PR DESCRIPTION
**Pull request by niharikak101**
Added a media query to scale the onboarding slides to fit a smaller window. Is this the correct approach, and does it look okay?

This is a (work in progress) fix for #2654

### CODE REVIEW

**Comment by johngruen** [comment]
@niharikak101 this looks good! I'm gonna merge and do one more PR to make the fonts slightly smaller. type fits in english, but i'm concerned about whether it would fit in other locales.

Thanks!

**Comment by niharikak101** [comment]
@johngruen Thank you! And for fixing the lint errors!

File: static/css/onboarding.scss [hunk 1]
```diff
@@ -259,3 +261,36 @@ body {
...
+    margin-left: - ($slide-width-small / 2) + 20;
+    margin-top: ($slide-height-small / 2) - 3;
+  }
+
+  #prev {
+    margin-left: - ($slide-width-small / 2) - ($next-prev-size / 2);
+}
+
+  #next,
+  #done, {
```
Comment (johngruen) [inline static/css/onboarding.scss]: @niharikak101 i didn't notice this at the time, but you have an extra comma on this line! I fixed it in the subsequent PR

Comment (johngruen) [inline static/css/onboarding.scss]: Oh, you already commented! Cheers!

Comment (niharikak101) [inline static/css/onboarding.scss]: @johngruen sorry about that.. should've paid more attention.

**Pull request merged**